contributors.html

```diff
@@ -9,6 +9,7 @@
 		<title>100Devs Statistics</title>
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
```

Comment:
Is there a reason to use the `shortcut` link type here? [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types) says it's non-conforming and doesn't suggest to use it anymore

Reply:
After doing a bit more digging, I see it's used for older Internet Explorer versions, but I don't think ppl using this site would be using Internet Explorer (and using an older version of it too). I don't think we should prioritize it so I'm in favor of removing `shortcut`, but open to hear your/anyone's thoughts on this

Reply:
I don't care for supporting internet explorer personally. I'm not sure the graphs would work on it, and I don't want to spend the time to make it work for ie

Reply:
Also the favicon doesn't have a transparent background 
![image](https://user-images.githubusercontent.com/78604367/161663363-417c3fe8-62de-4fa1-8216-b4f3f70f02f6.png)

Reply:
So should I change icon , i mean to say that should I change favicon

Reply:
> 

U r not wrong , but for this i will need various sizes of favicon img like 16×16 ,32×32 ,ico and svg and so on.

Reply:
Ok but I try another way in which we have to add .ico .png and apple touch icon ..